src/database/parse-server.ts

```diff
@@ -0,0 +1,48 @@
+import { DatabaseConfig } from "./database-config";
+
+export default class ParseServer implements Database {
+  private static instance: Database;
+
+  private Parse: any;
+  private Event: any;
+
+  public static getInstance(Parse: any, AsyncStorage: any): Database {
```

Comment:
Initially, I wanted this to just be `getInstance()` without any parameters (and just import Parse and AsyncStorage in this file), but that made it very difficult to unit test. This will only be called from App.tsx once for initialization anyway.

Components can get the DB instance like so: `import { Database } from "../../../App"`

And begin using it immediately like so: `Database.getEvents(10, 1, "dateTime");`